foundry.toml

```diff
@@ -3,7 +3,7 @@ src = 'src'
 out = 'out'
 
 optimizer = true
-optimizer_runs = 800
+optimizer_runs = 100
```

Comment:
Not sure if we actually want to check this in or not. I left it in anyway and we can undo if needed.

Reply:
would try reduce contract size down so this value can be set higher